package.json

```diff
@@ -60,7 +60,7 @@
     "solidity-coverage": "^0.5.11",
     "solium": "^1.1.8",
     "truffle": "5.0.0-next.17",
-    "truffle-keystore-provider": "^1.0.2",
-    "truffle-assertions": "0.6.3"
+    "truffle-assertions": "0.6.3",
```

Comment:
Can this change in this file and in `package-lock.json` be reverted since we are no longer introducing a new dependency in this PR?

Reply:
I think npm did this automatically (alphabetical order).

Reply:
fixed package-lock 35f5b36

but I think we can keep this change as it is convention to order dependencies alphabetically ?